.github/workflows/release.yaml

```
@@ -0,0 +1,30 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - main
```

```suggestion
on:
  push:
    branches:
      - main
      - release-*
    tags:
      - v.*
```

Maybe, we'd like to publish the page when we cut a new release.

---

Thanks! Fixed it.